chainweb.openapi.yaml

```diff
@@ -682,7 +672,8 @@ components:
       description: JSON Encoded Block Header
       required: [creationTime, parent, height, hash, chainId, weight, featureFlags, epochStart, adjacents, payloadHash, chainwebVersion, target, nonce]
       example:
-        $ref: '#/components/examples/blockHeaderPage/items/0'
+        value:
+          $ref: '#/components/examples/blockHeaderPage/value/items/0'
```

Comment:
This is the actual legal syntax for examples, they need a `value` and optionally a `summary`, see https://swagger.io/docs/specification/adding-examples/